tests/test_coordinator.py

```diff
@@ -73,14 +73,14 @@ async def test_coordinator_workflow(self):
         self.assertFalse(coordinator.need_rejoin(subscription.subscription))
 
         tp_list = subscription.assigned_partitions()
-        self.assertEqual(tp_list, set([('topic1', 0), ('topic1', 1),
-                                       ('topic2', 0), ('topic2', 1)]))
+        self.assertEqual(tp_list, {('topic1', 0), ('topic1', 1),
+                                       ('topic2', 0), ('topic2', 1)})
```

Comment:
https://github.com/aio-libs/aiokafka/pull/752/checks?check_run_id=2832624476#step:9:12
You can run `make lint` locally to see all linting errors

Reply:
Fixed in 642fcb13eb0ff2c59665ff117b66f84c167f621b